controllers/apartments.rb

```diff
@@ -10,5 +11,13 @@
 
 # show
 get '/apartments/:id' do
+  @apartment = Apartment.find(params[:id])
   erb :"apartments/show"
 end
+
+
+post "/apartments" do
+  new_apartment = Apartment.create(params[:apartment])
```

Comment:
Recommend `create!` to ensure it raises an error if there is a problem, instead of quietly swallowing the error.